net/http/src/THttpServer.cxx

```diff
@@ -614,6 +607,16 @@ void THttpServer::ProcessRequests()
    }
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Method called when THttpServer cannot process request
+/// By default such requests replied with 404 code
+/// One could overwrite with method in derived class to process all kinds of such non-standard requests
+
+void THttpServer::MissedRequest(THttpCallArg *arg)
```

Comment:
Please take a `THttpCallArg&` as this function requires it to be non-null.

Reply:
I have all other methods with `THttpCallArg *`, should I use here other signature?
If we want change interface, it should be changed with `shared_ptr<THttpCallArg>`